docs/docs/use-cases/flux.md

```diff
@@ -0,0 +1,189 @@
+│       └── kustomize.yaml
+└── clusters
+    └── dev
+        ├── flux-system
+        │   ├── gotk-components.yaml
+        │   ├── gotk-sync.yaml
+        │   └── kustomize.yaml
+        ├── podtato-head-source.yaml
+        └── podtato-head-kustomization.yaml
+```
+
+The `apps` directory contains application manifests, that will be deployed.
+The `clusters` directory contains Flux configuration manifests and custom
+resources, that are needed for the delivery.
+`apps` and `clusters` directories can live in two separate repositories,
+but for simplicity of this excercise, we will keep them in a single one.
+
+## Set up environment
+
+Before we start, you need to install Flux CLI and Keptn.
+You can find the installation instructions of Keptn [here](./../installation/index.md)
+and for Flux [here](https://fluxcd.io/flux/installation/).
+
+After successfully installing Flux CLI and Keptn, you need to
+retrieve your git repository credentials.
+You can use any available git providers, but be sure to store your `token`
+for later usage.
+For simplicity, we will use `GitHub`.
+
+As another step, you need to install Flux to your cluster.
+This step will require
+[bootstrapping the git repository](https://fluxcd.io/flux/installation/bootstrap/)
+in order to set up all needed Flux structures in the repository.
+For that, you can use the following command:
+
+```bash
+GITHUB_USER=<user-name> GITHUB_TOKEN=<token> \
+flux bootstrap github \
+  --owner=$GITHUB_USER \
+  --repository=podtato-head \
+  --branch=main \
+  --path=./clusters/dev \
+  --personal
+```
+
+The bootstrap command above does the following:
+
+* Creates a git repository `podtato-head` on your `GitHub` account.
+* Adds Flux component manifests to the repository -
+creates `./clusters/dev/flux-system/*` structure.
+* Deploys Flux Components to your Kubernetes Cluster.
+* Configures Flux components to track the path `./clusters/dev/` in the repository.
+
+## Creating application
+
+Now it's time to add the application together with pre- and
+post-deployments checks to the repository.
+
+Firstly, clone the `podtato-head` repository to your local machine.
+
+Add the following manifests into `podtato-head.yaml` representing the application
+and store it into `./apps/dev/` directory of your repository:
+
+```yaml
+{% include "./assets/flux/app.yaml" %}
+```
+
+Additionally, create a `kustomize.yaml` file right next to it:
+
+```yaml
+{% include "./assets/flux/kustomize.yaml" %}
+```
+
+You can commit and push these manifests to the git repository.
+
+> **Note**
+Notice, that the application has pre- and post-deployment tasks defined
+in the manifests.
+This enhances the
+[Flux pre and post-deployment jobs](https://fluxcd.io/flux/use-cases/running-jobs/)
+setup with added observability out of the box.
+
+## Set up continuous delivery for the application
+
+Firstly, we need to create a
+[GitRepository](https://fluxcd.io/flux/components/source/gitrepositories/)
+Flux custom resource
```

Comment:
```suggestion
Firstly, we need to create a Flux
[GitRepository](https://fluxcd.io/flux/components/source/gitrepositories/)
resource
```
You argued persuasively that we should just call these "resources" rather than "custom" resources for Keptn so I'm thinking the same standard applies to Flux resources.